### DISCUSSION TITLE
2.3.70 Errors detected during installation - Message log

### DISCUSSION THREAD

**Discussion by Farcocoop** [Security-Onion-Solutions/securityonion#5336] [General]
Hello, reading the sosetup.log:
 Errors detected during setup; skipping post-setup steps to allow for analysys of failures.
===========================
Stopping mysql
===========================
s-mysql
so-mysql
| I | Removing so-setup permission entry from sudores file

In file sosetup.log, searching for thestring "Error" I found:

[...] "id":"soadmin","haskey":true,"hasPassword":false}{"type":"NotFoundError","message"... 

And below:

Server is not ready Waiting fo value TheHive at http://xx.xxx.xxx_9000/thehive

Any idea? Thank you.

**Comment by dougburks** [comment]
Please provide more information as described in https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

- Install source. Did you install from our Security Onion ISO image or did you perform a network installation?
- If network install, did you install from CentOS 7 or Ubuntu 18.04?
- Install type. ex. eval, standalone, forward node, search node, etc
- Does so-status show all services running?
- Do you get any failures when you run salt-call state.highstate from the node?
- Does the grid status show any failures?
- Explain your issue. For example: Installation fails when I select this series of options...
- Provide applicable logs. For example: if you are having problems during soup, provide /root/soup.log
- If you need to attach files, please make sure they are plain text format. No Word docs or PDFs please.
